src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperContentExtensions.cs

```diff
@@ -51,7 +52,18 @@ public static class TagHelperContentExtensions
 
```

Comment:
Don't want to transition `Link` and `Script` tags to use `TagHelperOutput` since writing out their content as is is as performant as we could ever hope for :smile:. So if we stick with the writing the tags inline I'd prefer to nuke the method and move the logic into the `Script`/`Link` `TagHelper`s